pkg/peertls/io_util.go

```
@@ -54,11 +49,7 @@ func writeKey(key *ecdsa.PrivateKey, path string) error {
 		return errs.New("unable to open \"%s\" for writing", path, err)
 	}
 
-	defer func() {
-		if err := file.Close(); err != nil {
-			log.Printf("Failed to close file: %s\n", err)
-		}
-	}()
+	defer func() { _ = file.Close() }()
```

why are you ignoring the error ?

---

same